test/Meta_Test_Suite_Tests/data/asserts_outside_of_tests/Main.enso

```diff
@@ -0,0 +1,18 @@
+from Standard.Base import all
```

Comment:
How to test something that can't run inside of a `Test`, that's the problem!

Radek, looks like you decided to create whole new project to test the `clue`. I guess it'd be way more lighter to do something like [PrintTest](https://github.com/enso-org/enso/blob/develop/engine/runtime/src/test/java/org/enso/interpreter/test/PrintTest.java) - e.g. use JUnit to execute the test.

Reply:
As far as I know this project existed before `PrintTest` was created, here I just added one more test case to it.

I'm happy to convert it into the JUnit approach, but I think it should be done as a separate task and scheduled accordingly, especially as I'm not sure it's a high priority refactor for right now.

Reply:
But overall I agree, I think that instead of re-using the Test framework to test _itself_, using a separate test framework (JUnit) sounds safer, so I think it may be worth to do this switch at some point.

Do you want to create a ticket so that we can keep track of this change?

Reply:
No, I don't consider such a ticket necessary. There is a lot of new stuff. I didn't notice the project existed.